lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -387,6 +388,16 @@ def test_active_at_datetime(self):
         self.verification_active_at_datetime(attempt)
 
 
+class ManualVerificationTest(TestVerification):
+    """
+    Tests for the ManualVerification model
+    """
+    def test_active_at_datetime(self):
+        user = UserFactory.create()
+        attempt = ManualVerification.objects.create(user=user)
```

Comment:
Manual verification will be done by admin's so we can rename the variable `attempt` to just `manual_verification`